ui/app/templates/components/oidc/scope-form.hbs

```diff
@@ -78,40 +80,25 @@
   {{/if}}
 </form>
 
-<Modal
```

Comment:
styling changes are intentional, the floating copy button (with no toolbar) was anti-pattern
<img width="795" alt="Screenshot 2023-09-27 at 5 59 16 PM" src="https://github.com/hashicorp/vault/assets/68122737/ad8ee9b6-e6ff-483f-938f-9c1622189253">